docs/actions/gender.update.md

```diff
@@ -0,0 +1,15 @@
+## Payload
+```js
```

Comment:
I used it to see the highlighting changes as it is being done in some other md files. Since this is mostly not used in our docs I agreed with @rrenkert to delete these tags from the files I'm editing.